packages/@ama-sdk/schematics/schematics/typescript/core/index.ts

```diff
@@ -23,6 +23,36 @@ import { treeGlob } from '../../helpers/tree-glob';
 import { NgGenerateTypescriptSDKCoreSchematicsSchema } from './schema';
 import { OpenApiCliGenerator } from '../../code-generator/open-api-cli-generator/open-api-cli.generator';
 
+const JAVA_OPTIONS = ['specPath', 'specConfigPath', 'globalProperty', 'outputPath'];
+const OPEN_API_TOOLS_OPTIONS = ['generatorName', 'output', 'inputSpec', 'config', 'globalProperty'];
```

Comment:
```suggestion
const OPEN_API_TOOLS_OPTIONS = ['output', 'inputSpec', 'config', 'globalProperty'];
```

Reply:
This is required otherwise we will receive a log stating that `generatorName` won't be taken into account